userspace/libsinsp/threadinfo.h

```diff
@@ -113,6 +113,14 @@ class SINSP_PUBLIC sinsp_threadinfo: public libsinsp::state::table_entry
 		return (m_tid == m_pid) || m_flags & PPM_CL_IS_MAIN_THREAD;
 	}
 
+	/*!
+	  \brief Return true if this thread belongs to a pid namespace.
+	*/
+	inline bool is_in_pid_namespace() const
+	{
+		return (m_tid == m_vtid) || m_flags & PPM_CL_CHILD_IN_PIDNS;
```

Comment:
```suggestion
		return (m_tid != m_vtid) || m_flags & PPM_CL_CHILD_IN_PIDNS;
```

Reply:
oh now that copy and paste error was just fitting :rofl: